x/farming/types/params.go

```diff
@@ -23,8 +23,11 @@ var (
 	DefaultFarmingFeeCollector    = sdk.AccAddress(address.Module(ModuleName, []byte("FarmingFeeCollectorAcc"))).String()
 	DefaultDelayedStakingGasFee   = sdk.Gas(60000) // See https://github.com/tendermint/farming/issues/102 for details.
 
-	StakingReserveAcc = sdk.AccAddress(address.Module(ModuleName, []byte("StakingReserveAcc")))
 	RewardsReserveAcc = sdk.AccAddress(address.Module(ModuleName, []byte("RewardsReserveAcc")))
+
+	// ReserveAccAddressType is Address type of reserve accounts for staking, rewards, It could be changed,
+	// details on https://github.com/tendermint/farming/issues/200
```

Comment:
```suggestion
	// ReserveAddressType is an address type of reserve accounts for staking or rewards. 
	// The module uses the address type of 32 bytes length, but it can be changed depending on Cosmos SDK's direction.
	// The discussion around this issue can be found in this link.
	// https://github.com/tendermint/farming/issues/200
```